deploy/example_catalog/cr-app-cdh632cm.json

```diff
@@ -0,0 +1,415 @@
+{
+    "apiVersion": "kubedirector.hpe.com/v1beta1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "cdh632"
+    },
+
+    "spec" : {
+        "systemdRequired" : true,
+        "defaultPersistDirs" : ["/usr", "/opt", "/var", "/data"],
```

Comment:
@omhpe 
I missed this in our discussions. We need to do some more persistence tests and see whether we can remove some of these from this list.

Reply:
Would you rather merge this PR and then work on revising this app, or postpone the merge?

Reply:
Yes we would rather merge this now. And then revisit this.